app/controllers/resources_controller.rb

```diff
@@ -59,20 +59,47 @@ def check_permission
     privilege = params[:privilege]
     raise ArgumentError, "privilege" unless privilege
 
-    result = assumed_role.allowed_to?(privilege, resource)
+    begin
+      result = assumed_role.allowed_to?(privilege, resource)
+
+      write_success_audit(privilege, result)
+    rescue
+      write_failure_audit(privilege)
```

Comment:
please change this to `audit_failure` to match the rest of the places we do this

Reply:
don't we want to write the error that occurred? this is what we do in the authentication audit - https://github.com/cyberark/conjur/blob/493fc0379e2b3575ed011875d62b0ec11eda85d4/app/domain/authentication/authenticate.rb#L89